src/views/Viewer.vue

```diff
@@ -111,7 +118,8 @@ import ActionButton from '@nextcloud/vue/dist/Components/ActionButton'
 import isFullscreen from '@nextcloud/vue/dist/Mixins/isFullscreen'
 import isMobile from '@nextcloud/vue/dist/Mixins/isMobile'
 import Modal from '@nextcloud/vue/dist/Components/Modal'
-
+import Vue from 'vue'
```

Comment:
```
/home/runner/work/viewer/viewer/src/views/Viewer.vue
  121:7  error  Parsing error: Identifier 'Vue' has already been declared

```

Reply:
It I remove it, I get: 
521:6  error  'Vue' is not defined  no-undef